features/appmgt/org.wso2.carbon.appmgt.store.feature/src/main/resources/store/controllers/logout.jag

```diff
@@ -1,42 +1,44 @@
 <%
 (function(){
+    include("/jagg/jagg.jag");
+    var ssoIdpProvider = jagg.module('manager').getAPIStoreObj();
     var user = session.get("LOGGED_IN_USER");
     if(user === null) {
         response.sendRedirect('/store');
     } else {
         var config = require('/config/store.js').config(),
-                sso = require('sso'),
-                process = require("process"),
-                sso_sessions = application.get('sso_sessions'),
-                sessionId = session.getId(),
-                encodedSAMLLogoutRequest = sso.client.getEncodedSAMLLogoutRequest(user, sso_sessions[session.getId()], config.ssoConfiguration.issuer),
-                relayState = '/store',
-                serverHost = process.getProperty('server.host'),
-                postUrl = config.ssoConfiguration.identityProviderURL;
+            sso = require('sso'),
+            process = require("process"),
+            sso_sessions = application.get('sso_sessions'),
+            sessionId = session.getId(),
+            encodedSAMLLogoutRequest = sso.client.getEncodedSAMLLogoutRequest(user, sso_sessions[session.getId()],
+                config.ssoConfiguration.issuer),
+            relayState = '/store',
+            serverHost = process.getProperty('server.host'),
+            postUrl = ssoIdpProvider.getIdentityProviderUrl();
 
         var log = new Log();
         log.debug("store session index : " + sso_sessions[session.getId()]);
 
         %>
-        <div>
-           <p>You are now redirected to Stratos Identity. If the
-           redirection fails, please click the post button.</p>
+            <div>
+                <p>You are now redirected to Stratos Identity. If the
```

Comment:
** Just a thought: Not an issue** I think we need to change references "Stratos Identity" to "Identity Server" in all logout.jag files